src/scripts/turnmarker.js

```diff
@@ -5,44 +5,44 @@ import { Settings } from './settings.js';
 import { renderUpdateWindow } from './updateWindow.js';
 import { firstGM, Flags, FlagScope, socketAction, socketName } from './utils.js';
 
-let animator;
-let markerId;
+
 let lastTurn = '';
 
 
-Hooks.on('ready', async () => {
+Hooks.once('init', () => {
```

Comment:
What's the difference between 'ready' and 'init'. There's not really much docs on the hooks, and I'm still new to this.

I'm assuming 'init' is fired on initialization of everything, and ready is fired when the scene/map/browser is ready to render?

Reply:
This more or less like this:

1. `"init"` is called directly after `game` was created. This the earliest hook.
2. Then comes `"setup"` directly before `game` is setup (entities, UI, canvas, ...)
4. Then `"canvasInit"` after the canvas is created.
5. Then `"canvasReady"` after the canvas and its layers are drawn.
6. And finally `"ready"` when everything else is ready as well (keyboard, sockets, ...).

All these hooks are called once except `"canvasInit"`/`"canvasReady"`, which are also called every time the scene is switched.

For more details look at the `DOMContentLoaded` listener, and follow the code from there.